src/components/Button/Button.js

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+import propTypes from "prop-types";
+
+export const Button = ({ to, children, type, styleClass, onClick }) => {
```

Comment:
Button `API` is very limited. To be consistent I would prefer call passed styles from parent `classes`. The default button `type` should be set. If it will be `button` or `submit` the decision belongs to you, but probably `button` will be better. If `to` has been passed, the button should be represented by `Link` component. Buttons can be `disabled` also so there is a missing prop. Going the same way, the `disabled` props should be optional but `onClick` should be required.